lib/hiredis/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hiredis
-  VERSION = "0.6.1"
+  VERSION = "0.7.0"
```

Comment:
Can you remove the version bump. When the release happens is up to the project maintainers.